main.py

```diff
@@ -1787,7 +1787,7 @@ async def dark_market(message):
     cataine_prices = [[10, "Fine"], [30, "Fine"], [20, "Good"], [15, "Rare"], [20, "Wild"], [10, "Epic"], [20, "Sus"], [15, "Rickroll"],
                       [7, "Superior"], [5, "Legendary"], [3, "8bit"], [4, "Professor"], [3, "Real"], [2, "Ultimate"], [1, "eGirl"], [100, "eGirl"]]
 
-    if get_cat(message.guild.id, message.user.id, "cataine_active") == 0:
+    if get_cat(message.guild.id, message.user.id, "cataine_active") == 0 or get_cat(message.guild.id, message.user.id, "cataine_active") < int(time.time()):
```

Comment:
0 *is* below current time, this becomes unneeded
```suggestion
    if get_cat(message.guild.id, message.user.id, "cataine_active") < int(time.time()):
```